fourth-wall.css

```diff
@@ -38,7 +38,7 @@ body {
 }
 
 .holder{
-    padding-bottom: 40px;
+    padding-bottom: 2.5rem;
```

Comment:
My previous comment was querying the inconsistency rather than asking you to make the change. This file still has a mixture of `px` and `rem` values for margins and padding after this change so the situation isn't any better and I have no more info on why there's a mix than I did yesterday.

The reason I'm asking is that there's often a relationship between `font-size` and the amount of whitespace in a design. So if you make a change to the font size you often have to update the whitespace (margins/padding/line heights/etc.). Intelligent use of `em's` and `rem's` can mean that you just make one change to a font-size rather than font size + all the whitespace rules.

If the mixture of `px` and `rem` units hasn't got much thought behind it we should put some work on the backlog to address it to make the Fourth Wall less expensive to change.

Reply:
Thanks for the feedback @adoublebarrel very help and makes things a lot clearer I'll sort out the inconsistencies and the logic behind the configurable URL minimum approvals as well 👍